eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/InteractiveTxTlv.scala

```diff
@@ -70,6 +74,15 @@ sealed trait TxAckRbfTlv extends Tlv
 object TxRbfTlv {
   /** Amount that the peer will contribute to the transaction's shared output. */
   case class SharedOutputContributionTlv(amount: Satoshi) extends TxInitRbfTlv with TxAckRbfTlv
+
+  /** Amount that the peer will push to the other side when building the commitment tx (used in splices). */
+  case class PushAmountTlv(amount: MilliSatoshi) extends TxInitRbfTlv with TxAckRbfTlv
+
+  case class SpliceOutOutputTlv(output: TxOut) extends TxInitRbfTlv with TxAckRbfTlv
```

Comment:
Why do you need that? I don't think we need to exchange this beforehand, this will simply be added with `tx_add_output` by our peer during `interactive-tx`.

Reply:
This is related to https://github.com/ACINQ/eclair/pull/2584#discussion_r1087548380.